lib/types/items.d.ts

```diff
@@ -0,0 +1,64 @@
+import { IRazorpayQuery } from "./api";
+
+export interface IRazorpayItem {
```

Comment:
<!-- Suggestions propose improvements to the current subject. It's important to be explicit and clear on what is being suggested and why it is an improvement. Consider using patches and the blocking or non-blocking decorations to further communicate your intent.  -->
**suggestion:** ‏Lets make this internal and rename to `IRazorpayItemBaseRequestBody`.

Then, lets create two implementations, `IRazorpayItemCreateRequestBody extends IRazorpayItemBaseRequestBody`
and `IRazorpayItemUpdateRequestBody extends IRazorpayItemBaseRequestBody`

Reply:
Renamed these interfaces inside namespace